en/app.ftl

```diff
@@ -62,6 +62,18 @@ nav-profile-contact = Contact us
 nav-profile-contact-tooltip = Get in touch about { -brand-name-relay-premium }
 nav-profile-image-alt = { -brand-name-firefox-account(capitalization: "uppercase") } Avatar
 
+# Mobile menu icons 
```

Comment:
They're not icons (or alt text for icons), they're the actual menu items, correct?

Reply:
@flodolo Super sorry for the delay on this. We finally got the final list of strings and correct - these strings are for actual text links. The only two that aren't text links are `menu-toggle-open` and `menu-toggle-close` - these are for the toggle menu, they don't render actual text, but used for the `aria-label` and `title` of the SVG component. See attached image: 

<img width="1145" alt="image" src="https://user-images.githubusercontent.com/3924990/170608341-193bd2a4-094e-49d4-9f1b-8d8a9c98b360.png">

The 2 other strings are for the AppPicker component, I'm assuming we can deprecate the original strings at a future date?

 
<img width="381" alt="image" src="https://user-images.githubusercontent.com/3924990/170608517-2033e3f1-d6ab-4b78-8651-8773425ee92b.png">